.changeset/six-poets-love.md

```diff
@@ -0,0 +1,7 @@
+---
+'@compiled/eslint-plugin': patch
+---
+
+Adds `jsx-pragma` rule,
```

Comment:
Formatting is a bit weird here?

Reply:
This is a personal preference 😄 - you can have sentences that end with a comma or full stop and start the next work on a new line and it stays within the paragraph. I like this because it makes diffs and reviewing easier than reading a really long sentence left to right